### PR TITLE
Imos file name check

### PR DESCRIPTION
**Pull request by fxmzb123**

### CODE REVIEW

File: compliance_checker/imos/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
-from compliance_checker.imos.imos import *
\ No newline at end of file
+from compliance_checker.imos.imos import *
+from compliance_checker.imos.file_name_check import *
```
Comment (mhidas) [inline compliance_checker/imos/__init__.py]: Could you add a newline at the end of the last line?

File: compliance_checker/suite.py [hunk 2]
```diff
@@ -350,7 +350,7 @@ def load_dataset(self, ds_str):
                 raise StandardError("Unrecognized XML root element: %s" % xml_doc.tag)
         else:
             # no doc? try the dataset constructor
-            ds = Dataset(ds_str)
+            ds = Dataset(ds_str)
```
Comment (mhidas) [inline compliance_checker/suite.py]: @fxmzb123 Please remove trailing whitespace.

**Comment by fxmzb123** [comment]
@mhidas whitespace has been removed.

**Comment by fxmzb123** [comment]
@mhidas Newline has been added.

**Comment by mhidas** [comment]
Thanks @fxmzb123 , looks good.

**Pull request merged**